projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job.py

```diff
@@ -49,12 +49,14 @@ def __init__(
         job_name: str,
         team_name: str,
         rest_api_url: str,
+        arguments: dict = None,
         timeout: int = 5,  # TODO: Set reasonable default
         **kwargs,
     ) -> None:
         self.job_name = job_name
         self.team_name = team_name
         self._rest_api_url = rest_api_url
+        self.arguments = arguments
```

Comment:
should be private . All fields here should be marked as private per https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard#clarification-on-access-modifiers-in-python

Reply:
Thanks! Changed those 4 to private.